Assets/Scripts/CameraZone.cs

```diff
@@ -31,6 +33,18 @@ private void OnTriggerEnter(Collider other)
     {
         if (other.CompareTag(Constants.TAG_PLAYER))
         {
+            if (isFirst && isRestricted) //to be removed when alex finds where he calls it
+            {
+                StartCoroutine(UIManager.Instance.staminaBar.Glow());
+            }
+            if (isRestricted)
+            {
+                UIManager.Instance.staminaBar.FadeOut();
```

Comment:
I think it would make more sense if "FadeIn" meant that it was visible right? Maybe the names should be switched.